arm-asazure/2016-05-06/swagger/asazure.json

```diff
@@ -0,0 +1,479 @@
+      "patch": {
+        "description": "Updates the current state of the specified Analysis Services server",
+        "tags": [ "Servers" ],
+        "operationId": "Servers_Update",
+        "parameters": [
+          {
+            "name": "analysisServerName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Name of the Analysis Services server"
+          },
+          {
+            "name": "updateServerParameters",
+            "in": "body",
+            "required": true,
+            "schema": { "$ref": "#/definitions/AnalysisServicesResourceUpdateParameters" },
+            "description": "Request object for updating the server"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": { "$ref": "#/definitions/AnalysisServicesResource" }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AnalysisServices/servers": {
+      "get": {
+        "description": "Gets all the Analysis Services servers for the given resource group",
+        "tags": [ "Servers" ],
+        "operationId": "Servers_ListByResourceGroup",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": { "$ref": "#/definitions/AnalysisServicesResources" }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.AnalysisServices/servers": {
+      "get": {
+        "description": "List all the Analysis Services servers for the given subscription",
+        "tags": [ "Servers" ],
+        "operationId": "Servers_List",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": { "$ref": "#/definitions/AnalysisServicesResources" }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Resource": {
+      "description": "Represents an instance of Analysis Services resource",
```

Comment:
Comment for other reviewers:

> "Resource" does not have a property named "type". Instead the [AnalysisServicesResource has the property resourceType](https://github.com/Azure/azure-rest-api-specs/pull/551/files#diff-e01dd80d66ba6b11274fb3fbd9d09ecdR270). 

This is a deviation from the standard pattern and we have allowed this as an exception. The service team will make this change in the next iteration as they have other high priority stuff to finish right now.